crp_APL/crp_msgs/msg/EgoStatus.msg

```diff
@@ -0,0 +1,3 @@
+std_msgs/Header header
+
+float32 wheel_status
```

Comment:
Changing this to steering_wheel_angle.

Reply:
Fixed with [this commit](https://github.com/jkk-research/CooperativeResearchPlatform/commit/e03cdbba84f53ab37e72d2f03d6fdd50e607f087)

Reply:
Changed it back to road wheel angle because this message is used before the environmental fusion as an extender for the odometry message in CIL. It should contain all the extra information necessary for the Ego message.